scripts/appbuilder

```diff
@@ -172,7 +179,7 @@ export_image()
     docker export $TEMP_INAME -o $TEMP_FILE
     docker rm $TEMP_INAME >/dev/null
     if [ -z $DELETE_IMAGE ]; then
-	docker rmi -f $IMAGE_NAME
+	    docker rmi -f $IMAGE_NAME
```

Comment:
Please revert the bogus change.

Reply:
The first whitespace was a tab that made it look skewed. Converted it to a space now